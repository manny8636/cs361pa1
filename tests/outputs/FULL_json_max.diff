--- outputs/FULL_json_max.txt	2022-02-13 13:37:53.251613280 -0500
+++ expected/FULL_json_max.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,5 @@
+Key-value pairs:
+{
+KEYS[long long min] = -9223372036854775807
+}
 Success!
