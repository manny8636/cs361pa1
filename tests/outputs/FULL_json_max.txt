Parsing ../data/json_max.txt failed
