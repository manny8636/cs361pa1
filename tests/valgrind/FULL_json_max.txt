==3062916== Memcheck, a memory error detector
==3062916== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062916== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062916== Command: ../json -o ../data/json_max.txt
==3062916== 
Success!
==3062916== 
==3062916== HEAP SUMMARY:
==3062916==     in use at exit: 43 bytes in 1 blocks
==3062916==   total heap usage: 4 allocs, 3 frees, 16,899 bytes allocated
==3062916== 
==3062916== LEAK SUMMARY:
==3062916==    definitely lost: 43 bytes in 1 blocks
==3062916==    indirectly lost: 0 bytes in 0 blocks
==3062916==      possibly lost: 0 bytes in 0 blocks
==3062916==    still reachable: 0 bytes in 0 blocks
==3062916==         suppressed: 0 bytes in 0 blocks
==3062916== Rerun with --leak-check=full to see details of leaked memory
==3062916== 
==3062916== For lists of detected and suppressed errors, rerun with: -s
==3062916== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
