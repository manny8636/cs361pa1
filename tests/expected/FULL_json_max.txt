Key-value pairs:
{
KEYS[long long min] = -9223372036854775807
}
Success!
